src/styles/main.css

```diff
@@ -0,0 +1,69 @@
+.page__body {
+  margin: 0;
+  height: 100vh;
+  background-color: #fff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
```

Comment:
Code Style issue: Similar to the previous comment, consider using a CSS variable for this color.